PowerEditor/src/NppIO.cpp

```diff
@@ -352,15 +352,18 @@ BufferID Notepad_plus::doOpen(const generic_string& fileName, bool isRecursive,
             if (globbing)
             {
                 const TCHAR * substring = wcsrchr(fileName.c_str(), TCHAR('\\'));
-                size_t pos = substring - fileName.c_str();
+				if (substring)

```

Comment:
If needed some message can be shown in `else` case of [line](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/4986/files#diff-76c62ef652708b4b7b02f049a16b0629R391). Like "File is not available for opening" or something similar, but that is optional.

**But at any cost app should not crash, so this `null` checking is MUST.**